src/tests/Button.test.js

```diff
@@ -1,14 +1,41 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import Enzyme, { shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { shallow, render, mount } from 'enzyme';
 import Button from '../components/Button/Button';
 
-Enzyme.configure({ adapter: new Adapter() });
 
 describe('<Button />', () => {
+  const button = shallow(<Button />);
+
+
   it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Button />, div);
+    render(button);
+  });
+
+  it('have a button', () => {
+    button.find("component-button");
+    expect(button).toHaveLength(1);
+  });
+
+  it("have a name prop", () => {
+    const props = {name:'9'};
```

Comment:
Good Prop Checks, Sometimes it is important to verify that our components are able to receive props well. This is just as important as expecting them to function properly, and on larger applications can become a nightmare if not done at this small a scale first.